packages/twitter-verifier-app/pages/MainPage.tsx

```diff
@@ -126,26 +124,27 @@ export const MainPage: React.FC<{}> = (props) => {
     }));
   };
 
-  const reformatProofForChain = (proof: string) => {
+  const reformatProofForChain = (proofStr: string) => {
+    if (!proofStr) return [];
+    
+    const proof = JSON.parse(proofStr);
+
     return [
-      proof ? JSON.parse(proof)["pi_a"].slice(0, 2) : null,
-      proof
-        ? JSON.parse(proof)
-            ["pi_b"].slice(0, 2)
-            .map((g2point: any[]) => g2point.reverse())
-        : null,
-      proof ? JSON.parse(proof)["pi_c"].slice(0, 2) : null,
-    ];
+      proof.pi_a.slice(0, 2),
+      proof.pi_b.slice(0, 2).map((s: string[]) => s.reverse()).flat(),
+      proof.pi_c.slice(0, 2),
+    ].flat();
   };
 
   const { config } = usePrepareContractWrite({
-    addressOrName: import.meta.env.VITE_CONTRACT_ADDRESS, // TODO: get address
-    contractInterface: abi, // TODO: get abi
+    address: "0x76E5978bdfA5Da926eCE0cE23F7A07648C525b5F", // TODO: get address
```

Comment:
wait why is this hard coded and not an env var or smth

Reply:
Updated this 👍

Reply:
oh wait maybe we shouldn't push .envs to git lol, prob better to make this a .env.example and default to this if .env doesnt exist, so that people are careful, since they usually expect .env to be gitignored?

Reply:
.env is  git ignored by default - I added this explicitly.
For the frontend .env serve like a constants file - the contents of that are copied to places using them during builds. The values are anyway exposed in the browser.
We can rename this to .env.production and .env.local (I do this in some other projects) but the effect is same.